### ISSUE TITLE
Focus state on multiline selection

### ISSUE THREAD

**Issue by henryjameslau** [ONSvisual/accessibility#31]
![image](https://user-images.githubusercontent.com/2945099/93232214-8b615b80-f771-11ea-8935-9b3cef4d46ca.png)
Focus state around the X on some colour will fail contrast ratios

**Comment by henryjameslau** [comment]
![image](https://user-images.githubusercontent.com/2945099/93232731-25c19f00-f772-11ea-92fc-c20cb09dd563.png)

Suggest using outline and box-shadow
```
.search-choice-close:focus {
    outline: orange 3px solid;
    box-shadow: 0 0 0 6px black;
}
```
